util/dvsim/FlowCfg.py

```diff
@@ -550,7 +574,7 @@ def _publish_results(self):
 
         # Publish the results page.
         # First, write the results html file temporarily to the scratch area.
-        results_html_file = self.scratch_path + "/results_" + self.timestamp + \
+        results_html_file = self.report_path + "/publish_results_" + self.timestamp + \
```

Comment:
add prefix `publish_` to distinguish with `results_{time-date}.html`.
This publish html file also contains a URL list of previous published path.